docs/configuration.rst

```diff
@@ -42,9 +42,129 @@ File: config.yml
 
 This is the core configuration file that defines most of `Cowbird`'s data configuration which it must work with to
 manage :term:`Services` components.
-A basic `config.example.yml`_ file is provided, for sample definition of expected schemas per service. Please refer to
-its comment header for specific format and parameter
-details.
+A basic `config.example.yml`_ file is provided, for sample definition of expected schemas per service.
+
+This config file contains the following required sections :
+
+services:
+#########
+
+The ``services`` section contains the definition of managed services by Cowbird.
+Each service is provided as a string that must match an actual implementation in Cowbird.
+Each service must be further configured with one or more of the following parameters. If
+a required parameter is missing for a service it will throw a ``ServiceConfigurationException`` exception.
+
+.. code-block:: yaml
+
+  service:
+    active:             [optional, default=False] bool allowing to deactivate a service and stop managing it
+    priority:           [optional, default=math.inf] Relative priority between services while handling events
+                        Lower values have higher priority, default value is last
+    url:                [optional, default=None] URI of the web service represented by this Cowbird service.
+                        Some Cowbird services do not represent web service, but other will throw an exception if
+                        missing.
+    workspace_dir:      [optional, default=None] Location of the users workspace root.
+                        Required for the following services : FileSystem, Catalog and Geoserver
+
+sync_permissions:
+#################
+
+This section defines how to synchronize permissions between services when they share resources.
+This is used only for the synchronization of permissions between Magpie services/resources.
+The ``sync_permissions`` are defined first by a list of Magpie services and their associated resources,
+defined in the ``services`` section below. The mappings defining how the resources should be synchronized are described
+in the other section ``permissions_mapping``.
```

Comment:
Could reference the actual section with 
```rst
:ref:`permissions_mapping`
```
(might need to remove the `:` in the section header)